### PR TITLE
FAB improvements

### PR DESCRIPTION
**Pull request by micahmo**
This PR polishes several issues identified with the new FAB feature.

- Fix issue not seeing "New Post" when navigating to community from sidebar
 - Do not show "Subscriptions" from a pushed community page (where the sidebar is not visible)
 - Show main post action ("Reply") in extended actions to match community behavior
 - Fix issue where FAB could get stuck open, for example
   - Open in post, then navigate back
   - Open in community, then navigate back
   - Open in main feed, then navigate to inbox/saved post
   > Note: I would still like to make `isFabOpen` more scoped, but it turns out that's a pretty big refactor

> Remember to review with Hide Whitespace selected.

### CODE REVIEW

**Comment by shortwavesurfer2009** [comment]
If this is a FAB rework can accessibility be addressed in this one too? As in #517.

File: lib/post/pages/post_page.dart [hunk 1]
```diff
@@ -87,225 +87,216 @@ class _PostPageState extends State<PostPage> {
...
                               children: [
-                                Icon(
-                                  Icons.warning_rounded,
-                                  color: theme.colorScheme.errorContainer,
+                                ActionButton(
+                                  onPressed: () {
+                                    HapticFeedback.mediumImpact();
+                                    replyToPost();
+                                  },
+                                  title: "Reply to Post",
```
Comment (hjiangsu) [inline lib/post/pages/post_page.dart:143]: Localization!

**Review by hjiangsu — COMMENTED** [review]
It looks good! Just added comments on where we can potentially add in localization!

**Comment by micahmo** [comment]
Alright, localization and accessibility should now be all set!

**Comment by hjiangsu** [comment]
Looks good! All thats left is the merge conflicts and it should be good to merge :D

**Comment by micahmo** [comment]
> All thats left is the merge conflicts

Ironically, these conflicts were created by my own other PR. 😆 Fixed now.

**Comment by hjiangsu** [comment]
Looks good! Thanks :D

**Pull request merged**